src/dotnet-aspnet-codegenerator/dotnet-aspnet-codegenerator.csproj

```diff
@@ -1,5 +1,5 @@
-<Project Sdk="Internal.AspNetCore.Sdk">
-
+﻿<Project Sdk="Microsoft.Net.Sdk">
+    <Import Project="$(RepositoryEngineeringDir)\Package.props" />
```

Comment:
All projects that generated packages should import this and common properties should move to this.